src/usr/local/buildpack/tools/v2/python.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# sets the correct shebang for python
+fix_python_shebangs() {
+  # https://github.com/koalaman/shellcheck/wiki/SC2044
+  while IFS= read -r -d '' file
+  do
+    case "$(head -1 "${file}")" in
+    "#!"*"/bin/python" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python:" "${file}"
+      ;;
+    "#!"*"/bin/python${MAJOR}" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python${MAJOR}:" "${file}"
+      ;;
+    "#!"*"/bin/python${MAJOR}.${MINOR}" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python${MAJOR}.${MINOR}:" "${file}"
+      ;;
+    esac
+  done < <(find "${versioned_tool_path}/bin" -type f -exec grep -Iq . {} \; -print0)
+}
+
+function install_tool () {
+  local versioned_tool_path
+  local file
+  local url
+  local arch
+  local version_codename
+
+  # get semver -> major, minor, patch
+  check_semver "${TOOL_VERSION}"
```

Comment:
Why do we need this again?

Reply:
In this case MAJOR, MINOR and PATCH are unset, beause they get only exported in the requirements check.
If you have a better idea, let me know.
We can also move it one level up, but that might not make sense for all tools

Reply:
🤔 i think those variables are always availabe after  `check_semver` call, because they are explict exported

https://github.com/containerbase/buildpack/blob/00ec26c2c88bcc59c7b0fddeded9da296140f56f/src/usr/local/buildpack/util.sh#L66-L68

Soo i thinnk we don't need to call again

Reply:
I can remove that line, but my feeling is that it won't work then. 
Let me do that

Reply:
Let me check if I did anything wrong

Reply:
I did indeed